src/shared/features.js

```diff
@@ -0,0 +1,45 @@
+export const FEATURES = [
+    {
+        id: 0,
```

Comment:
we could probably add some font-awesome icon constants here as well.

We will need to make a constants folder, which I can do for us.